base/rounding.jl

```diff
@@ -117,6 +120,8 @@ arithmetic functions ([`+`](:func:`+`), [`-`](:func:`-`), [`*`](:func:`*`), [`/`
 and [`sqrt`](:func:`sqrt`)) and type conversion.
 
 See [`RoundingMode`](:obj:`RoundingMode`) for available modes.
+
+**Warning**: This feature is still experimental, and may give unexpected or incorrect values.
```

Comment:
I think there's a specific admonition syntax for this with `!!!` now, there should be a few other similar examples. also run `make docs` and commit the rst